test/jdk/java/nio/channels/Channels/TransferTo.java

```diff
@@ -23,6 +23,10 @@
 
 import static java.lang.String.format;
 
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertThrows;
+import static org.testng.Assert.assertTrue;
```

Comment:
Static imports in tests are usually grouped by convention after regular imports.

Reply:
I assume this is a OpenJDK convention? Because all IDEs I used so far sorted them *before* regular imports. I will fix that.

Reply:
Done by https://github.com/openjdk/jdk/pull/5421/commits/3301a86d4381d1d12a57cbbe57010f58cafef084.